### PR TITLE
Comments and likes

### PR DESCRIPTION
**Pull request by uryelah**
- Added comments and likes features
- Added db seeds to the project
- Added model and integration tests for comments and likes
- Implemented selenium driver on capybara for confirmation alert test

*EDIT:*
If the integration tests are not passing for you try the instructions below :
  - go to spec/spec_helper and comment out line 99: ```Capybara.default_driver = :selenium```
  - then disable this test: spect/integration/delete_comment_spec.spec

### CODE REVIEW

**Comment by uryelah** [comment]
The tests failing were due to the selenium driver, we added instructions under EDIT at the pull request first comment to make it run if that issue happens.
If the selenium driver works in your system the expected behavior should be a system alert asking if you allow it to run in the system, at the first time running integration tests only, then selenium opening a headless firefox browser and doing the integration tests there.

File: app/views/posts/_post.html.erb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 <div>
   <%= content_tag(:p, "#{post.author.first_name} #{post.author.last_name}") + content_tag(:p, "#{post.content}") %>
-</div>
\ No newline at end of file
+  <%= render 'comments/form', post: post %>
+  <p> <%= post.likes.count %> users liked this post! </p>
+  <%= like_or(post) %>
```
[the remark this answers is not in the document]
Comment (uryelah) [reply app/views/posts/_post.html.erb]: Collection rendering applied

[the remark this answers is not in the document]
Comment (uryelah) [reply app/views/posts/_post.html.erb]: Amazing sugestion! Thank you :-)

**Comment by samgaco** [comment]
Hi @uryelah and @jairjy :high_brightness: 

You can move on, the milestone is approved. Try to add the warning about running capybara with selenium in the subsequent pull requests.

Regards!

-------------------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**